client/app/scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -3,8 +3,14 @@
 
 'use strict';
 
-MetadataCtrl.$inject = ['$scope', 'desks', 'metadata', '$filter', 'privileges', 'datetimeHelper'];
-function MetadataCtrl($scope, desks, metadata, $filter, privileges, datetimeHelper) {
+MetadataCtrl.$inject = [
+    '$scope', 'desks', 'metadata', '$filter', 'privileges', 'datetimeHelper',
+    'preferencesService'
+];
+function MetadataCtrl(
+    $scope, desks, metadata, $filter, privileges, datetimeHelper,
+    preferencesService
+) {
```

Comment:
I refer to the paranthesis being on their own lines, didn't see this style in the code base till now.